foodsaving/conversations/tests/test_tasks.py

```diff
@@ -0,0 +1,25 @@
+from django.core import mail
+from django.test import TestCase
+from django.utils import timezone
+
+from foodsaving.conversations.tasks import notify_participants
+from foodsaving.groups.factories import GroupFactory
+from foodsaving.users.factories import VerifiedUserFactory, UserFactory
+
+
+class TestConversationNotificationTask(TestCase):
```

Comment:
I wrote this test before noticing there was some related ones inside the api test file, which did not seem the right place for them. This is about running the task function itself, so maybe it's seperate...